src/cmd-run

```diff
@@ -158,7 +170,11 @@ EOF
         exit 1
     fi
 fi
-set -- -fw_cfg name=opt/com.coreos/config,file="${IGNITION_CONFIG_FILE}" "$@"
+
+coreos_gf_run_mount "${VM_DISK}"
+coreos_gf upload ${IGNITION_CONFIG_FILE} /boot/config.ign
```

Comment:
This should be `/boot/ignition/config.ign`.

Reply:
Doh...